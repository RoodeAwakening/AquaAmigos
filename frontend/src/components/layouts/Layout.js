import React from 'react';


function Layout({ children }) {
  return (
    <div className="layout__container">

      <div className="layout__container__nav">
        <h1>nav will be here</h1>
      </div>

      <div className="layout__container__content">
        {children}
      </div>

      <div className="layout__container__sidebar">
        <h1>sidebar here </h1>
      </div>


    </div>
  );
}

export default Layout;
