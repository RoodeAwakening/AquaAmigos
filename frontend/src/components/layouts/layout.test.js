import React from 'react';
import { shallow, configure } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import Layout from './Layout';

configure({ adapter: new Adapter() });
global.confirm = () => true;

function setup() {
  return shallow(<Layout />);
}

describe(' Layout via Enzyme', () => {
  let layout;

  beforeEach(() => {
    layout = setup();
  });

  afterEach(() => {
    layout.unmount();
  });

  it(' Should render layout', () => {
    expect(toJson(layout)).toMatchSnapshot();
  });
});
