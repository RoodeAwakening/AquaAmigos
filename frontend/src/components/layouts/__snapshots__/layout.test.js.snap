// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Layout via Enzyme  Should render layout 1`] = `
<div
  className="layout__container"
>
  <div
    className="layout__container__nav"
  >
    <h1>
      nav will be here
    </h1>
  </div>
  <div
    className="layout__container__content"
  />
  <div
    className="layout__container__sidebar"
  >
    <h1>
      sidebar here 
    </h1>
  </div>
</div>
`;
